adapter-venice/src/main/java/io/nosqlbench/adapter/venice/util/VeniceAdapterMetrics.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2022-2023 nosqlbench
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.nosqlbench.adapter.venice.util;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Timer;
+import io.nosqlbench.api.config.NBLabeledElement;
+import io.nosqlbench.api.config.NBLabels;
+import io.nosqlbench.api.engine.metrics.ActivityMetrics;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class VeniceAdapterMetrics implements NBLabeledElement {
```

Comment:
NB5 is introducing a new labeling framework. The current way of S4J adapter metrics implementation actually will cause labeling conflict.

I made some changes yesterday to fix this. Please see this: https://github.com/nosqlbench/nosqlbench/pull/1271/files#diff-31f09e589d4bae88649200fccc05520e05835c7da263aad54cedebf08d4b5d90

Reply:
You can also see the current NB Kafka and NB Pulsar adapter metrics and they have the right approach.

Reply:
Rebased and fixed